### DISCUSSION TITLE
Feature Request: Blacklist for Words or Phrases

### DISCUSSION THREAD

**Discussion by DiStopian** [AlexW00/obsidian-note-linker#58] [Ideas]
Hi Alex,

As mentioned in my previous _feature request_ about the "Select All" option, I had other suggestions.

I would like to see a blacklist for specific words or phrases, for example "Introduction" often shows up on the list but refers to different topics. If there were a way to blacklist it so I don't have to worry about it slipping through.

Thank you.

V.

**Comment by axamaxa** [comment]
yes, Alex; this feature is almost crucial to the successful deployment of the plugin. There _must_ be some way to exclude those pesky words (en masse) that have no meaningful contribution to a note but occur almost ubiquitously in every note (like the word 'note', or the  word 'about' for example ....imagine writing a special note about 'about' and linking to it in every file). The sheer number of linkable innanities is almost transfinite: we _desperately_ need the blacklist feature!
